### DISCUSSION TITLE
Historical data for relative humidity?

### DISCUSSION THREAD

**Discussion by dsheehan312** [open-meteo/open-meteo#1067] [Q&A] [answered]
Hi all,

So, I am running the API locally and trying to find a way to download historical relative humidity data (25 years). It doesn't appear to be in the era or era land data. I see that its available in the example api on the website, but I was thinking maybe the data was downloaded/imported via other means. Worst case I can find an alternate source and import into the DB that I am using for modeling but would rather to keep all the data in one location, and essentially, I need roughly 1500 counties in the USA (so downloading via csv won't work).  Any help would be appreciated!

**Comment by patrick-zippenfenig** [comment, marked as answer]
Hi, relative humidity is calculated on-demand from `dew_point_2m` and `temperature_2m`. You need both for the API to return humidity.

**Reply by dsheehan312** [reply]
Thanks that worked! Didn't know about the dew point
